run.zsh

```diff
@@ -102,9 +120,10 @@ get_avg_startup() {
   local startup_time startup_total startup_avg
   local i n
 
-  startup_times=($(cut ${results_dir}/${1}.log -c49-53))
+  startup_times=($(sed -e 's/.*cpu //' -e 's/ total//' ${results_dir}/${1}.log))
```

Comment:
Suppose this could still be safer. Using characters is suspect because the CPU % could take up a variable amount of space.